lib/lib-dynamodb/src/commands/utils.ts

```diff
@@ -90,12 +90,41 @@ const processAllKeysInObj = (obj: any, processFunc: Function, keyNodes: KeyNodes
   }, {} as any);
 };
 
+function copyWithoutFunctions(o: any, depth = 0): any {
```

Comment:
we can actually remove all the other places that handle `convertClassInstanceToMap` since this is exhaustive

Reply:
not many other places where we handle that option, can open a new PR for it if we decide to remove the checks for that option